### ISSUE TITLE
A3:We PR and Merging before set the action up

### ISSUE THREAD

**Issue by JunyangPan** [CISC-CMPE-327/Information#79]
I am sorry about this, we forgot to set the action up at first, so some of our group member write the test code, PR and Merging without action.

Then now I set the action up and it failed.

so I begin to fix the problem and it passed now, in this case we have a new PR link which PR link should we hand in on onq? Both of them?

**Comment by steven-hh-ding** [comment]
okay it is fine then. just submit all. I assume that each PR (before your fix) did pass locally? But in the future, never merged PRs before you have all the proper check setup/passed.

**Comment by JunyangPan** [comment]
> okay it is fine then. just submit all. I assume that each PR (before your fix) did pass locally? But in the future, never merged PRs before you have all the proper check setup/passed.

Yes, they passed locally but after we merging it did not pass. I sorry about this, this will not happen again
